src/main/java/com/commercetools/sync/inventories/utils/InventoryUpdateActionUtils.java

```diff
@@ -0,0 +1,112 @@
+package com.commercetools.sync.inventories.utils;
+
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+import io.sphere.sdk.inventory.commands.updateactions.ChangeQuantity;
+import io.sphere.sdk.inventory.commands.updateactions.SetExpectedDelivery;
+import io.sphere.sdk.inventory.commands.updateactions.SetRestockableInDays;
+import io.sphere.sdk.inventory.commands.updateactions.SetSupplyChannel;
+import io.sphere.sdk.models.Reference;
+
+import javax.annotation.Nonnull;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+import static com.commercetools.sync.commons.utils.CommonTypeUpdateActionUtils.buildUpdateAction;
+
+/**
+ * This class provides static utility methods for building update actions related to inventories.
+ */
+public final class InventoryUpdateActionUtils {
+
+    private InventoryUpdateActionUtils() {
+        throw new AssertionError();
+    }
+
+    /**
+     * Compares the {@code quantityOnStock} values of an {@link InventoryEntry} and an {@link InventoryEntryDraft}
+     * and returns an {@link Optional} of update action, which would contain the {@code "changeQuantity"}
+     * {@link UpdateAction}. If both {@link InventoryEntry} and {@link InventoryEntryDraft} have the same
+     * {@code quantityOnStock} values, then no update action is needed and empty optional will be returned.
+     * If the {@code quantityOnStock} from the {@code newEntry} is {@code null}, the new {@code quantityOnStock} will
+     * have a value of 0L.
+     *
+     * @param oldEntry the inventory entry that should be updated
+     * @param newEntry the inventory entry draft which contains new quantity on stock
+     * @return optional containing update action or empty optional if quantities on stock are identical
+     */
+    @Nonnull
+    public static Optional<UpdateAction<InventoryEntry>> buildChangeQuantityAction(@Nonnull final InventoryEntry
+                                                                                           oldEntry,
+                                                                                   @Nonnull final InventoryEntryDraft
+                                                                                       newEntry) {
+        final Long oldQuantityOnStock = oldEntry.getQuantityOnStock();
+        final Long newQuantityOnStock = newEntry.getQuantityOnStock() == null ? Long.valueOf(0L) : newEntry
```

Comment:
I see, however, I'd use NumberUtils.LONG_ZERO constant for that to save couple processor cycles on each invocation.
This constant is calculated only once during class loading instead of doing it every time in this method.